app/lib/bk/compat/parsers/gha/expressions.rb

```diff
@@ -154,6 +166,16 @@ def self.replace_context_needs(path)
         end
       end
 
+      def self.replace_context_github(var_name)  
+        translated_var = @github_bk_context_mapping[var_name]
+        case translated_var
+        when String
+          "$#{translated_var}"
+        when nil
+          raise NoMethodError
```

Comment:
I think this should instead print a value reporting to the user that it is not supported instead of raise an error

Reply:
the `NoMethodError` exception is caught in the calling method and prints out the error we currently see saying that it can not be translated. it is an interesting idea but I probably agree that we should customize the error message

Reply:
Counted for this as below 👇